### ISSUE TITLE
Compatibility with Samsung Galaxy S6 Edge+

### ISSUE THREAD

**Issue by citytrader** [PkmX/lcamera#178]
I will be glad if you can make this camera compatible with Samsung Galaxy S6. here is the dump:
- The screen has a black rectangle, looks like smaller than screen resolution
- It will be nice if you can support 10s exposure of stock camera

Media Dump (I made a mistake, correct name of pastebin should be Samsung Galaxy S6 Edge Plus)
http://pastebin.com/u7FXyBcJ

**Comment by PkmX** [comment]
Can you paste the output on something like pastebin or GitHub's gist and post the link back here? The unformatted output makes it quite hard to follow.

**Comment by citytrader** [comment]
I edited the post with pastbin link, let me know if its ok!

**Comment by citytrader** [comment]
I'm a pro photographer so if you need to ask more questions, your are welcome!
